nsv13/code/modules/overmap/traders_items.dm

```diff
@@ -166,17 +175,40 @@
 		/obj/item/ship_weapon/parts/loading_tray,\
 	)
 
+/datum/trader_item/vls_circuit
+	name = "VLS tube circuit board"
+	desc = "The critical component for expanding your missile complement!"
+	price = 5000
+	stock = 10
+	unlock_path = /obj/item/circuitboard/machine/vls
+	special_requirement = 100 //At least put some effort into it
+
+/datum/trader_item/firing_electronics
+	name = "Firing Electronics"
+	desc = "Essential electronics for building most modern naval weapons."
+	price = 20000
+	stock = 2
+	unlock_path = /obj/item/circuitboard/machine/vls
```

Comment:
This should probably be firing electronics, not a VLS board like the sold item above.

Reply:
Yep... that's pretty essential